website/docs/docs/deploy/ci-jobs.md

```diff
@@ -70,26 +77,9 @@ If you're not using dbt Cloud’s native Git integration with [GitHub](/docs/cl
       - `non_native_pull_request_id` (for example, BitBucket)
    - Provide the `git_sha` or `git_branch` to target the correct commit or branch to run the job against. 
 
-## Example pull requests
```

Comment:
cleaned this section up a little bit, page getting a bit long. moved it up to lines 50-58 to be closer to the "set up" section. i think just one example is sufficient to illustrate the integration. 

lemme know what you think! happy to revert